pmm-app/tests/qan_test.js

```diff
@@ -148,15 +150,83 @@ xScenario('Open the QAN Dashboard and check Details -> Tables', async (I, adminP
+xScenario('Open the QAN Dashboard and show tooltip @new-qan', async (I, adminPage, qanPage) => {
   I.amOnPage(qanPage.url);
   qanPage.showTooltip(4, 1);
   I.wait(5);
   I.see('dasdadadad');
   // TODO: check changes in url
 });
+
+Scenario('Open the QAN Dashboard and check existence of filters @new-qan', async (I, qanPage) => {
+  qanPage.verifyFiltersSectionIsPresent();
+});
+
+Scenario('Open the QAN Dashboard and check work of button "reset all" @new-qan', async (I, qanPage) => {
+  qanPage.selectFilter(qanPage.fields.filterCheckboxSelector);
+  qanPage.resetAllFilters();
+  I.dontSeeCheckboxIsChecked(qanPage.fields.filterCheckboxSelector);
+});
+
+Scenario('Open the QAN Dashboard, add a column and make sure that this option not available anymore @new-qan', async (I, qanPage) => {
+  qanPage.addColumn('Bytes Sent');
+  qanPage.verifyColumnIsNotAvailable('Bytes Sent');
+});
+
+Scenario('Open the QAN Dashboard and verify that hovering over a time metric displays a tooltip with a graph @new-qan', async (I, qanPage) => {
+  const ROW_NUMBER = 1;
+  const QUERY_TIME_COLUMN_NUMBER = 3;
+
+  qanPage.showTooltip(ROW_NUMBER, QUERY_TIME_COLUMN_NUMBER);
+  I.seeElement(qanPage.elements.latencyChart);
+});
+
+Scenario('Open the QAN Dashboard and verify that hovering over a non-time metric displays a tooltip without a graph @new-qan', async (I, qanPage) => {
+  const ROW_NUMBER = 1;
+  const QUERY_COUNT_COLUMN_NUMBER = 2;
+
+  qanPage.showTooltip(ROW_NUMBER, QUERY_COUNT_COLUMN_NUMBER);
+  I.dontSeeElement(qanPage.elements.latencyChart);
+});
+
+Scenario('Open the QAN Dashboard and verify that the metric value matches the "Per sec" value in the tooltip. @new-qan', async (I, qanPage) => {
+  const ROW_NUMBER = 1;
+  const QUERY_COUNT_COLUMN_NUMBER = 2;
+  const queryCountColumnSelector = qanPage.summarizeLocator(ROW_NUMBER, QUERY_COUNT_COLUMN_NUMBER);
+
+  qanPage.showTooltip(ROW_NUMBER, QUERY_COUNT_COLUMN_NUMBER);
+  let qpsMetricValue = await I.grabTextFrom(queryCountColumnSelector);
+  let qpsTooltipValue = await I.grabTextFrom('[data-qa="metrics-list"] [data-qa="qps"] span');
+  assert.equal(qpsMetricValue.replace(/[^0-9.]/g,""), qpsTooltipValue.replace(/[^0-9.]/g,""));
+});
+
+Scenario('Open the QAN Dashboard and check that changing the time range clears the selected row. @new-qan', async (I, qanPage, adminPage) => {
+  I.waitForElement(qanPage.elements.tableRowSelector, 30);
+  I.forceClick(qanPage.elements.tableRowSelector);
+  adminPage.applyTimeRange('Last 3 hours');
+  I.dontSeeElement(qanPage.elements.selectedOverviewRow);
+  I.dontSeeElement(qanPage.elements.detailsSection);
+});
+
+Scenario('Open the QAN Dashboard and check that changing the time range resets current page to the first. @new-qan', async (I, qanPage, adminPage) => {
+  qanPage.paginationGoTo(2);
+  adminPage.applyTimeRange('Last 3 hours');
+  qanPage.waitForResponsePath('/v0/qan/GetReport');
```

Comment:
/v0/qan/GetReport path could be stored in qanPage.requests obj and called from there
insert await for verifySelectedPageIs